Makefile

```diff
@@ -92,7 +92,7 @@ $(TERRAFORM_PROVIDER_SCHEMA): $(TERRAFORM)
 	@$(INFO) generating provider schema for $(TERRAFORM_PROVIDER_SOURCE) $(TERRAFORM_PROVIDER_VERSION)
 	@mkdir -p $(TERRAFORM_WORKDIR)
 	@echo '{"terraform":[{"required_providers":[{"provider":{"source":"'"$(TERRAFORM_PROVIDER_SOURCE)"'","version":"'"$(TERRAFORM_PROVIDER_VERSION)"'"}}],"required_version":"'"$(TERRAFORM_VERSION)"'"}]}' > $(TERRAFORM_WORKDIR)/main.tf.json
-	@$(TERRAFORM) -chdir=$(TERRAFORM_WORKDIR) init > $(TERRAFORM_WORKDIR)/terraform-logs.txt 2>&1
+	@$(TERRAFORM) -chdir=$(TERRAFORM_WORKDIR) init -upgrade > $(TERRAFORM_WORKDIR)/terraform-logs.txt 2>&1
```

Comment:
@ccremer how do you usually handle output in Makefiles?

Print everything? Only print stderr?

Reply:
depends on the relevance and maybe also length, but I'm not really consistent either. when there's a `make` target for printing `helm template` as a form of manual verification, then obviously everything. `mkdir`'s,  `rm` or `sed`s not.
If the output is, I dunno, 20 lines but it's quite important because you run it regularly, then I'd display all output.
Basically, if you think it's worth printing everything, go ahead, otherwise logfiles are fine too.

Also worth including in that decision is GitHub actions. If something fails, it's gonna be hard to print from logfiles in a GitHub action run if the stderr doesn't reveal the actual error... (e.g. renovate etc)

Reply:
Thanks for the explanation. If there's no consistent way I'll leave it this way for now.